### PR TITLE
Test

### PR DESCRIPTION
**Pull request by zacck-zz**
#### What does this PR do?
Adds  a sample test case to the test suite as reference for adding unit tests later.
#### Description of Task to be completed?
Add a failing test to `tests` directory and ensure it is run when the test runner for the application is run.
#### How should this be manually tested?
Run `yarn test` / `npm test` to view the  failing test
#### Questions:
In order for us to add test coverage to the application we need to ensure that documentation exists for the application. The documentation will then be used to build tests and view which tests are failing or passing this will depend on  #32  being worked on.

### CODE REVIEW

File: tests/Sample.elm [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module Sample exposing (..)
```
Comment (akabiru) [inline tests/Sample.elm:1]: @zacck Looks good. Please run [elm-format](https://github.com/AndelaOSP/shufflebox-frontend/pull/31#discussion_r110131496), it's our current formatting tool.

**Comment by akabiru** [comment]
@zacck Are you referring to inline documentation? 🤔

**Comment by zacck-zz** [comment]
No I wouldn't necessarily say so! Say some sort of checklist as to what the components should be able to do  under what scenarios.[
screen shot 2017-04-06 at 2 00 04 pm
](url)

If you can see the screenshot attached above as an example from one of my a description of a component and what it does to make sure the component are working correctly

**Comment by akabiru** [comment]
@zacck  Awesome. I get what you mean. Sort of like feature tests right? That'd be awesome!

**Comment by zack** [comment]
@akabiru @zacck ^^^

**Comment by zacck-zz** [comment]
Yes sir!! And In order to reduce the amount of errors and pointless code one should always write tests before writing the component. This may seem tedious but if you try it for a whole day I guarantee you will see why it is so vital!  You end up writing components that are already designed and described.

**Comment by akabiru** [comment]
@zacck We were kind of learning on the go 😅. You have a good point TDD is the way to go.

**Comment by zacck-zz** [comment]
Let's go then ...

**Comment by akabiru** [comment]
@zacck Please address the formatting issue so that I can merge the PR.

**Comment by zacck-zz** [comment]
@akabiru I ran it!

**Pull request merged**